22_pipeslab/pipeslab.txt

```diff
@@ -3,19 +3,106 @@ For each of the following, type a single command line that performs:
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
+maxim@maxim-virtual-machine:~$ ls -l | grep '^...x' | cut -d' ' -f3,9
+maxim Desktop
+maxim Documents
+maxim Downloads
+maxim lab
+maxim Music
+maxim Pictures
+maxim Public
+maxim Templates
+maxim tmp
+maxim Videos
+maxim wildcards
+
+
+  
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
   Answer: 
 
+  maxim@maxim-virtual-machine:~$ ls -l | grep '^...x..x..x' | cut -d' ' -f3,9
+maxim Desktop
+maxim Documents
+maxim Downloads
+maxim lab
+maxim Music
+maxim Pictures
+maxim Public
+maxim Templates
+maxim tmp
+maxim Videos
+maxim wildcards
+
+  
   3. Print the number of lines from "ls -l" which represent paths that were modified today (no matter which day we run the command)
   Answer: 
+  
+maxim@maxim-virtual-machine:~$ ls -l | grep $(date | cut -d' ' -f3)
```

Comment:
the usage of ```cut``` here is very problematic here.. there is just too much that can break(for example on my machine -f4 works) .. 

I would use the format  string to date that you learned in chapter 1:
```date "+%b %e```
and actually call date twice, like so:
```
grep "$(date +%b)\s* $(date +%e)"
```

the ```\s* ``` part means at least one space

and then ```wc``` ;)